bridges/snowbridge/primitives/core/src/location.rs

```diff
@@ -27,7 +27,7 @@ pub type AgentId = H256;
 pub type AgentIdOf = HashedDescription<
 	AgentId,
 	(
-		DescribeHere,
+		DescribeTerminus,
```

Comment:
I'd pefer to not change it here. Please check https://github.com/vgeddes/polkadot-sdk/pull/28#issuecomment-2699997747

Let's move furthur discussions there. Though I think the cleanup is not a blocking issue, and I prefer to move it out of the scope for V2.